csrankings-m.csv

```diff
@@ -500,6 +500,7 @@ Marcus V. dos Santos,Ryerson University,https://www.scs.ryerson.ca/~m3santos,NOS
 Marcus Vinícius Alvim Andrade,Universidade Federal de Viçosa,http://www.dpi.ufv.br/~marcus,NOSCHOLARPAGE
 Marek Chrobak,Univ. of California - Riverside,http://www.cs.ucr.edu/~marek,RutwCU4AAAAJ
 Marek Cygan,University of Warsaw,http://www.mimuw.edu.pl/~cygan,df8TSy4AAAAJ
+Marek Eliáš,Bocconi University,http://elias.ba30.eu,hX2T3SQAAAAJ

```

Comment:
Need to add an entry for other alias (see https://dblp.org/pid/173/4575.html).

Reply:
The other DBLP entry with the same name is a different person.

Reply:
No, I mean: "Marek Eliás 0001" and "Marek Eliáš". You need both entries.

Reply:
The two entries must be:
* Marek Eliás 0001 
* Marek Eliáš
You have now added "Marek Eliáš 0001", which is not a DBLP alias.

Reply:
Ok, I added the alias.

Reply:
(See comment above)

Reply:
Sorry, hopefully it's ok now